### ISSUE TITLE
ErrorException on Silverstripe 5

### ISSUE THREAD

**Issue by hoekie** [sunnysideup/silverstripe-ecommerce#55]
Hi!

When I install this ecommerce addon on a fresh installation of Silverstripe 5, I get the following errorexception:

copy(C:\wamp64\www\silverstripe-ecommerce\vendor\sunnysideup\model-admin-many-tabs\client): Failed to open stream: No such file or directory

Do you know how to solve this issue?

Thanks!

Richard

**Comment by sunnysideup** [comment]
I will look into this and get back to you.

**Comment by hoekie** [comment]
Thanks in advance!

**Comment by sunnysideup** [comment]
This has now been fixed.  I am really keen to help you set up the e-commerce.  If you have any issues we will try to fix them for you ASAP.

**Comment by sunnysideup** [comment]
Make sure you do a  composer update. This is the issue that fixes it: https://github.com/sunnysideup/silverstripe-model-admin-many-tabs/commit/d6376f88737593ed5f44b73b7ba41c7df864ba23

**Comment by hoekie** [comment]
Perfect! Installing works like a charm!
I will setup the site later this week.

Thank you for your effort and fast response!

Richard

**Comment by sunnysideup** [comment]
@hoekie  - if you need any assistance then please let us know.  We are really keen to make it work for you.

**Comment by hoekie** [comment]
Oké, after installing and running dev/build I ran into the following:

1. 
Fatal error: Declaration of Sunnysideup\Ecommerce\Cms\CMSPageAddControllerProducts::doCancel($data, $form) must be compatible with SilverStripe\CMS\Controllers\CMSPageAddController::doCancel(array $data, SilverStripe\Forms\Form $form): SilverStripe\Control\HTTPResponse in C:\wamp64\www\mysite\vendor\sunnysideup\ecommerce\src\Cms\CMSPageAddControllerProducts.php on line 42

I just started with silverstripe so I don't know how to solve this myself...

2.
According to your docs, there should be an ecommerce.yml.example file in the _config folder to be copied into my own but it is nowhere to be found.
Or I'm looking on the wrong spot ;)

3.
The demo site is not working (http://www.silverstripe-ecommerce.com/)

So far my findings regarding the installation.

Should I open a new issue?

Cheers!

Richard
